src/Dashboard.tsx

```diff
@@ -24,17 +24,24 @@ function Dashboard() {
   const location = useLocation();
   const navigate = useNavigate();
 
+  const [isSessionRestore, setSessionRestore] = useState(false);
```

Comment:
What's this flag for?

Reply:
I also was able to remove this hook. Branch updated.